addons/web_editor/static/src/js/rte.js

```diff
@@ -533,9 +533,8 @@ var RTE = Widget.extend({
             }, 150); // setTimeout to remove flickering when change to editable zone (re-create an editor)
             this.$last = null;
         }
-        if ($editable.length && (!this.$last || this.$last[0] !== $editable[0]) &&
-                ($target.closest('[contenteditable]').attr('contenteditable') || "").toLowerCase() !== 'false') {
-
+        if (($editable.length && ($target.closest('[contenteditable]').attr('contenteditable') || "").toLowerCase() !== 'false' || dom.isImg($target[0]))
+            && (!this.$last || this.$last[0] !== $editable[0])) {
```

Comment:
Hello @dja-odoo 
My point is that this is not an issue, this is the intended behavior. If you put `contenteditable="false"` on a xml section, it is expected that the section content cannot be edited (note that I think that this case may not exist at all). In your video example, if you add on the image `contenteditable="true"`, then the image will be editable again (if I am not mistaken). So what I proposed here is to fix the slideshow snippet to still add dynamically `contenteditable="false"` but also automatically add `contenteditable="true"` on the inside images -> this should prevent the user to enter text but allow them to change the images in the snippet.